be/src/olap/olap_cond.cpp

```diff
@@ -537,7 +537,7 @@ bool CondColumn::eval(const segment_v2::BloomFilter* bf) const {
 
 Status Conditions::append_condition(const TCondition& tcond) {
     DCHECK(_schema != nullptr);
-    int32_t index = _schema->field_index(tcond.column_name);
+    int32_t index = _schema->field_index(tcond.column_unique_id);
```

Comment:
Should we hide index in the TabletSchema? We can use a function like this _schema->get_clolumn_by_unique_id(tcond.column_unique_id).